apps/mobile-wallet/src/contexts/SendContext.tsx

```diff
@@ -108,7 +108,7 @@ export const SendContextProvider = ({ children }: { children: ReactNode }) => {
       const data = await buildSweepTransactions(address, address.hash)
       setUnsignedTxData(data)
     } catch (e) {
-      showToast(getHumanReadableError(e, 'Error while building the transaction'))
+      showExceptionErrorToast(e, 'Could not build transaction')
```

Comment:
This would result in showing the title `Could not build transaction` and as a secondary text would be the potential stringified exception error.